firepit/clickhousestorage.py

```diff
@@ -0,0 +1,585 @@
+import logging
+import os
+import re
+import time
+import traceback
+
+import orjson
+import psycopg2
+import psycopg2.extras
```

Comment:
These are for postgresql; you don't need them, right?

Reply:
I have removed the  psycopg2 imports. orjson is still required.